### PR TITLE
Moved root history out of instance details and into standalone mgmt call

### PR DESCRIPTION
**Pull request by ed-**
Successfully excised root history and placed it into extensions where it belongs.

### CODE REVIEW

File: reddwarf/extensions/mgmt/views.py [hunk 1]
```diff
@@ -62,3 +68,18 @@ def data(self):
...
+
+
+class RootHistoryView(object):
+
+    def __init__(self, instance_id, enabled='Never', user_id='Nobody'):
+        self.instance_id = instance_id
+        self.enabled = enabled
+        self.user = user_id
+
+    def data(self):
```
Comment (ed-) [inline reddwarf/extensions/mgmt/views.py:80]: Figured I'd try this technique out once, as an alternative to writing the dictionary out manually.
tree() is autovivifying, so there's no need to define `res['root_history']` before mentioning it.

**Comment by cp16net** [comment]
+1

**Comment by rnirmal** [comment]
+1

File: reddwarf/extensions/mgmt/views.py (hunk 1 above)
Comment (hub-cap) [inline reddwarf/extensions/mgmt/views.py:80]: i like. i had no idea about the tree() function

**Pull request merged**